### ISSUE TITLE
Clicking on any element in the bar has no effect.

### ISSUE THREAD

**Issue by dhruvmachhi** [FelixKratz/SketchyBar#752]
I'm on MacOS Sequoia, I have Yabai and SketchyBar (w/ the default sketchybarrc). Clicking on the space buttons has no effect for me.

It turned out I had not given SketchyBar permissions in accessibility but even after enabling it and restarting both sketchybar and my computer it still doesn't work and now it's not even in my system preferences.

I used the command line sketchybar and found out it was due to my yabairc
